Updating dependencies to latest .NET Servicing

joperezr opened the pull request:
## Description

Updating .NEt Servicing to latest versions.

cc: @eerhardt 

## Checklist

- Is this feature complete?
  - [x] Yes. Ready to ship.
- Are you including unit tests for the changes and scenario tests if relevant?
  - [x] No
- Did you add public API?
  - [x] No
- Does the change make any security assumptions or guarantees?
  - [x] No
- Does the change require an update in our Aspire docs?
  - [x] No

joperezr committed:
Updating dependencies to latest .NET Servicing

eerhardt commented on eng/Versions.props:
```diff
@@ -43,10 +43,10 @@
     <!-- dotnet/extensions -->
     <MicrosoftExtensionsAIVersion>9.9.1</MicrosoftExtensionsAIVersion>
     <MicrosoftExtensionsAIPreviewVersion>9.9.1-preview.1.25474.6</MicrosoftExtensionsAIPreviewVersion>
-    <MicrosoftExtensionsHttpResilienceVersion>9.9.0</MicrosoftExtensionsHttpResilienceVersion>
```
How about the above versions?

```xml
    <!-- dotnet versions for running tests -->
    <DotNetRuntimePreviousVersionForTesting>8.0.20</DotNetRuntimePreviousVersionForTesting>
    <DotNetRuntimeCurrentVersionForTesting>9.0.9</DotNetRuntimeCurrentVersionForTesting>
    <!-- dotnet 8.0 versions for running tests - used for templates tests -->
    <DotNetSdkPreviousVersionForTesting>8.0.414</DotNetSdkPreviousVersionForTesting>
    <!-- dotnet 9.0 versions for running tests - used for templates tests -->
    <DotNetSdkCurrentVersionForTesting>9.0.305</DotNetSdkCurrentVersionForTesting>
```

joperezr commented on eng/Versions.props:
```diff
@@ -98,22 +98,22 @@
+    <MicrosoftExtensionsFileProvidersEmbeddedLTSVersion>8.0.21</MicrosoftExtensionsFileProvidersEmbeddedLTSVersion>
+    <MicrosoftAspNetCoreSignalRClientLTSVersion>8.0.21</MicrosoftAspNetCoreSignalRClientLTSVersion>
     <!-- Runtime -->
     <MicrosoftExtensionsHostingAbstractionsLTSVersion>8.0.1</MicrosoftExtensionsHostingAbstractionsLTSVersion>
```
I used the BarID that was released from runtime when calling the update-dependencies command, which would have updated any packages that had shipped.

joperezr committed:
Update .NET test runtime and SDK versions

Bump DotNetRuntimePreviousVersionForTesting to 8.0.21, DotNetRuntimeCurrentVersionForTesting to 9.0.10, DotNetSdkPreviousVersionForTesting to 8.0.415, and DotNetSdkCurrentVersionForTesting to 9.0.306 in eng/Versions.props.

eerhardt committed:
Fix DependencyInjection.AutoActivation reference
